app/assets/css/sass/_activity.scss

```diff
@@ -73,10 +73,20 @@
 
 }
 
-// remove border and padding
-#buddypress ul.rtmedia-list li {
-	border-bottom: 0;
-	padding-bottom: 0;
+#buddypress {
+    // Remove border and padding.
+    ul.rtmedia-list {
+        li {
+            border-bottom: 0;
+            padding-bottom: 0;
+        }
+    }
+    // Media title fix on legacy .
+    ul.activity-list {
+        li {
+            overflow: unset!important;
```

Comment:
@dharmin Can we avoid usage of `! important` here? If possible override by adding more specificity.

Reply:
BuddyPress has inline CSS control over it, so need to add `!important`.